src/models/chan.js

```diff
@@ -23,18 +23,21 @@ function Chan(attr) {
 	}, attr));
 }
 
-Chan.prototype.sortUsers = function() {
-	this.users = _.sortBy(
-		this.users,
-		function(u) { return u.name.toLowerCase(); }
-	);
-
-	["+", "%", "@", "&", "~"].forEach(function(mode) {
-		this.users = _.remove(
-			this.users,
-			function(u) { return u.mode === mode; }
-		).concat(this.users);
-	}, this);
+Chan.prototype.sortUsers = function(irc) {
+	var userModeSortPriority = {};
+	irc.network.options.PREFIX.forEach(function(prefix, index) {
```

Comment:
> It seems to me that sortUsers should be using serverOptions.PREFIX instead of options.PREFIX

They're the same thing. `serverOptions` exists because its an object in *our* network instance, and it gets synchronized to the frontend client for later use. I didn't want to pipe the whole irc-fw options/networks objects to the client, as they have a lot of data.